documentation/Language.md

```diff
@@ -1,14 +1,32 @@
+If you look at the raw file you can see which parts of the code don't compile. If a codeblock looks like this it compiles:
+```lpg
+// Something that works
+```
+and if it looks like this it does not work
+```
+// Something that does not compile
+```
+
 # Language overview
 
+1. [Hello World](#Hello-World)
 1. Types and syntax
     1. [Variables and constants](#Variables-and-constants)
     1. [Functions](#Function-Syntax)
     1. [Tuple](#Tuples)
     1. [Interfaces](#Interfaces)
-1. Structures
+1. Concepts
     1. [Match](#Match)
     1. [Loops](#Loops)
+    1. [Comments](#Comments)
 1. [Standard Library Functions](#Standard-library-functions)
+1. [Optimizations](#Optimizations)
+
+## Hello World
+The first thing in every programming language is the "Hello World"-program. This is the code you need to write in lpg.
+```
+print("Hello world")
```

Comment:
`print` was removed recently. There will be a replacement, but that will take a while.

Reply:
Removed